### ISSUE TITLE
Allow images in table editor cells

### ISSUE THREAD

**Issue by thecursedfly** [jaap-karssenberg/zim-desktop-wiki#432]
Using the following table editor example source:

```
| Column title 1 | Column title 2 | Icons                          |
|:---------------|:---------------|:-------------------------------|
| text 1         | text 2         | {{./icon1.png}}{{./icon2.png}} |
```

with the two icon images in the page subfolder

results in an error when visualizing the page in Zim ("Looks like you have found a bug").

When the wiki page is exported to html, the table is correctly displayed with the icons inside the table cell.

Error text:

This is zim 0.67
Platform: posix
Locale: it_IT UTF-8
FS encoding: UTF-8
Python: (2, 7, 14, 'final', 0)
Gtk: (2, 24, 32)
Pygtk: (2, 24, 0)
No bzr version-info found

======= Traceback =======
  File "/usr/lib/python2.7/site-packages/zim/gui/pageview.py", line 5232, in set_page
    self.set_parsetree(tree, template)
  File "/usr/lib/python2.7/site-packages/zim/gui/pageview.py", line 5340, in set_parsetree
    buffer.set_parsetree(tree)
  File "/usr/lib/python2.7/site-packages/zim/gui/pageview.py", line 696, in set_parsetree
    self.insert_parsetree_at_cursor(tree)
  File "/usr/lib/python2.7/site-packages/zim/gui/pageview.py", line 763, in insert_parsetree_at_cursor
    self._insert_element_children(root, raw=raw)
  File "/usr/lib/python2.7/site-packages/zim/gui/pageview.py", line 920, in _insert_element_children
    obj = ObjectManager.get_object('table', element.attrib, element)
  File "/usr/lib/python2.7/site-packages/zim/objectmanager.py", line 81, in get_object
    obj = factory(attrib, text)
  File "/usr/lib/python2.7/site-packages/zim/plugins/tableeditor.py", line 137, in create_table
    (header, rows) = self._tabledom_to_list(text)
  File "/usr/lib/python2.7/site-packages/zim/plugins/tableeditor.py", line 153, in _tabledom_to_list
    row = [ElementTree.tostring(r, 'utf-8').replace('<td>', '').replace('</td>', '') for r in row]
  File "/usr/lib64/python2.7/xml/etree/ElementTree.py", line 1126, in tostring
    ElementTree(element).write(file, encoding, method=method)
  File "/usr/lib64/python2.7/xml/etree/ElementTree.py", line 820, in write
    serialize(write, self._root, encoding, qnames, namespaces)
  File "/usr/lib64/python2.7/xml/etree/ElementTree.py", line 939, in _serialize_xml
    _serialize_xml(write, e, encoding, qnames, None)
  File "/usr/lib64/python2.7/xml/etree/ElementTree.py", line 932, in _serialize_xml
    v = _escape_attrib(v, encoding)
  File "/usr/lib64/python2.7/xml/etree/ElementTree.py", line 1092, in _escape_attrib
    _raise_serialization_error(text)
  File "/usr/lib64/python2.7/xml/etree/ElementTree.py", line 1052, in _raise_serialization_error
    "cannot serialize %r (type %s)" % (text, type(text).__name__)
TypeError: cannot serialize <UnixFile: /home/user/Notebooks/Notes/Wiki/Page/icon1.png> (type UnixFile)

**Comment by Beutlin** [comment]
Currently only text is supported within a table cell.
Probably that is the cause for this error.

**Comment by pverflow** [comment]
i had the same error thought i did something wrong. quite unfortunate that we cant put images into tables :(

**Comment by aloboa** [comment]
This is 4 years old... Any chance of adding this enhancement any time soon?

**Comment by introt** [comment]
No, I'm afraid no one has stepped up. For further inquiries on the status of tickets please refer to the Discussions
